SmartDeviceLink/SDLIAPSession.m

```diff
@@ -9,17 +9,17 @@
 #import "SDLMutableDataQueue.h"
 
 NS_ASSUME_NONNULL_BEGIN
-#define IO_STREAMTHREAD_NAME            @ "com.smartdevicelink.iostream"
-#define STREAM_THREAD_WAIT_SECS         1.0
+NSString *const ioStreamThreadName  = @ "com.smartdevicelink.iostream";
+NSTimeInterval const streamThreadWaitSecs = 1.0;
 
 @interface SDLIAPSession ()
 
 @property (assign, nonatomic) BOOL isInputStreamOpen;
```

Comment:
We don't strictly need it, but we do need to check that both streams are open before calling the delegate opened method. I didn't feel a lot of pressure to change the usage of those boolean member variables, but I suppose we could check the stream states directly.